docs/topics_subscribers.md

```diff
@@ -0,0 +1,332 @@
+    less-cli log --project my-less-project --path topics/user_created/send_welcome_email
+    less-cli log --project my-less-project --path topics/user_created/send_to_analytics
+    ```
+  </TabItem>
+
+</Tabs>
+
+```
+  POST /topics/{topic_id}
+```
+
+## Handling Failing Messages
+Less automatically retains failed messages and tries to process them again later.
+
+In order to see this in action we can force a crash in one of our subscribers:
+<Tabs groupId="programming-language" queryString="programming-language">
+  
+  <TabItem value="nodejs" label="Node.js">
+    ```bash
+    touch less/topics/user_created/send_welcome_email/index.js
+    ```
+    
+    ```js title="less/topics/user_created/send_welcome_email/index.js" showLineNumbers
+    exports.process = async (user) => {
+      throw new Error('This subscriber will fail to process messages.');
+      console.log(`Sending a welcome email to ${user.email}`);
+    };
+    ```
+  </TabItem>
+  
+</Tabs>
+
+Deploy the change and publish another message to your topic. You will notice that while the `user_create/send_welcome_email` subscriber is failing, the `user_create/send_to_analytics` subscriber is working just fine.
+
+Once we fix the crash and deploy we will notice that the previously failing message will be processed successfully.
+
+## Cross-Application Topics
+Less also makes it easy to subscribe to topics from different Less applications. This makes it easier than ever to build **distributed**, **fault-tolerant**, **event-driven** **microservices**.
+
+### Subscribe to External Topics
+
+1. Create an `external_topics` folder inside of `less` to set up your external app connections.
+```bash
+mkdir -p less/external_topics
+```
+
+2. Add the projects you want to connect to to the `external_topics` folder.
+```bash
+mkdir less/external_topics/my-less-project
+```
+
+3. Add the topic want to subscribe to to the respective project folder.
+```bash
+mkdir less/external_topics/my-less-project/user_created
+```
+
+4. Create your subscriber
+<Tabs groupId="programming-language" queryString="programming-language">
+  <TabItem value="nodejs" label="Node.js">
+    ```bash
+    touch less/external_topics/my-less-project/user_created/index.js
+    ```
+    
+    ```js title="less/external_topics/my-less-project/user_created/notify_friends/index.js" showLineNumbers
+    exports.process = async (user) => {
+      console.log('Telling all your friends that you have joined...');
+    };
+    ```
+  </TabItem>  
+</Tabs>
+
+5. Configure your project connection.
+Add the folder name to the `less.config` file as an environment variable. The value of this variable should match the name of the application you want to connect to.
+```yaml
+env_vars:
+  - my-less-project
+```
+
+Export the name of the application (deployment) you want to connect to. This allows you to point your application to different deployments (development, stage, production, etc.).
+```bash
+export my-less-project=my-less-project
```

Comment:
This is not a valid name format for a env var due the use of hyphen. And also the name for all the env var now should be in upper case, so the name of the env var on uppercase will match the name of the project